main.go

```diff
@@ -56,7 +57,7 @@ func main() {
 		fmt.Fprintf(os.Stderr, "cannot find binary %q in any of:\n", binary)
 		workspaces := filepath.SplitList(build.Default.GOPATH)
 		for i, workspace := range workspaces {
-			path := filepath.Join(workspace, "bin", binary)
+			path := filepath.Join(workspace, "bin", binaryName(binary))
```

Comment:
In terms of correctness, I think either `cannot find binary foobar.exe in any of C:\GOPATH\bin\foobar.exe` or `cannot find command foobar in any of C:\GOPATH\bin\foobar.exe` would be valid.

But not `cannot find binary `foobar` in any of C:\GOPATH\bin\foobar.exe`, since binary name is actually "foobar.exe" and not "foobar".